requirements.txt

```diff
@@ -1,5 +1,6 @@
 appdirs
 biopython
+# biosimulators-simularium
```

Comment:
is this a comment or some sort of requirements.txt syntax?

Reply:
This is a comment, but I just removed it as the implementation is different.